src/coreclr/nativeaot/System.Private.CoreLib/src/System/Threading/Timer.Windows.cs

```diff
@@ -22,7 +22,7 @@ private TimerQueue(int id)
         }
 
         [UnmanagedCallersOnly]
-        private static void TimerCallback(IntPtr instance, IntPtr context, IntPtr timer)
+        private static void TimerCallback(IntPtr instance, nint context, IntPtr timer)
```

Comment:
This is a Kernel32 P/Invoke that should probably just use `void*` since that's what native takes.